### PR TITLE
feat(table):  add "colSize" in fieldProps within proTable's columns to customize query form item width

### PR DESCRIPTION
**Pull request by LyndonXiao**
feat(table): 在columns中的fieldPorps对象中添加colSize,对应QueryFilter组件的colSize属性,以自定义查询表单项的宽度

### CODE REVIEW

File: packages/table/src/Form/index.tsx [hunk 1]
```diff
@@ -227,6 +230,7 @@ export const formInputRender: React.FC<{
       }}
       rules={undefined}
       key={`${item.dataIndex || ''}-${item.key || ''}-${item.index}`}
+      colSize={colSize}
```
Comment (chenshuai2144) [inline packages/table/src/Form/index.tsx:233]: 这个功能是谁支持的，form 的？

Comment (LyndonXiao) [reply packages/table/src/Form/index.tsx:233]: 具体我不清楚，只是看到demo文档中出现过
image
image

proForm的change log也有提到，所以是支持colSize的，只是没有proTable这边没有暴露出来

# 1.0.0-beta.6 (2020-07-24)

### Bug Fixes

- rename @ant-design/form -> @ant-design/pro-form ([#16](https://github.com/ant-design/pro-components/issues/16)) ([82e0b6d](https://github.com/ant-design/pro-components/commit/82e0b6d20decc4b391925d469b1c250c69ded485))
- secondary typescript define ([32f8571](https://github.com/ant-design/pro-components/commit/32f8571e4e1a8007f4208736cc0aec87548b1874))
- useForm not work problem ([c7a0dba](https://github.com/ant-design/pro-components/commit/c7a0dbaa99b9f5c32d37820420764662a3aeeddd))

### Features

- add new Radio form component ([#18](https://github.com/ant-design/pro-components/issues/18)) ([47c41aa](https://github.com/ant-design/pro-components/commit/47c41aa556972b0f7cbb51c164432f9f5bdbf9b0))
- Group titleRender support props ([1c2e669](https://github.com/ant-design/pro-components/commit/1c2e669a7041657b43332cc97068a4f9a4bd926b))
- init ProForm SearchFilter & LightFilter ([#15](https://github.com/ant-design/pro-components/issues/15)) ([1939b8d](https://github.com/ant-design/pro-components/commit/1939b8d5b76e14ce01f727304fbc75aabd46ca6e))
- QueryFilter support _**colSize**_ ([530b084](https://github.com/ant-design/pro-components/commit/530b084a7520a7d8aeb4e5e14f435455efe73ecb))
- QueryFilter support labelWidth ([6b7d13b](https://github.com/ant-design/pro-components/commit/6b7d13bda3b040c6ff4b344fdc49ff0be53cfd9f))
- support colSize ([70ab58b](https://github.com/ant-design/pro-components/commit/70ab58b8082ef87bd323a0b93871e649c02ae665))

Comment (chenshuai2144) [reply packages/table/src/Form/index.tsx:233]: soga，是我们自己支持的，我试试

**Comment by chenshuai2144** [comment]
ci 修一下

**Comment by LyndonXiao** [comment]
哭泣，要不咱自己人提交一下？我撤销这个PR

**Comment by chenshuai2144** [comment]
这是提升自己的好机会呀，一个ts 的错误而已

**Comment by LyndonXiao** [comment]
行

File: packages/table/src/Form/index.tsx [hunk 2]
```diff
@@ -136,6 +136,12 @@ export const formInputRender: React.FC<{
       ? (itemValueType({}, type) as ProFieldValueType)
       : itemValueType) as ProFieldValueType) || 'text';
 
+  const { onChange, colSize, ...restFieldProps } = getFieldPropsOrFormItemProps(
+    item.fieldProps || {},
```
Comment (chenshuai2144) [inline packages/table/src/Form/index.tsx:140]: as 一下吧。就是这里有问题

File: packages/table/src/Form/index.tsx [hunk 3]
```diff
@@ -136,6 +136,12 @@ export const formInputRender: React.FC<{
...
+    form,
+    item,
+  );
```
Comment (chenshuai2144) [inline packages/table/src/Form/index.tsx:143]:
```suggestion
  ) as any; 
```

**Comment by chenshuai2144** [comment]
懂了 demo 可以用 yarn test -u 来解决

**Comment by LyndonXiao** [comment]
下次知道了😁

**Pull request merged**